src/components/Cards.js

```diff
@@ -15,6 +15,7 @@ function Cards() {
               text="Explore southern Albania and the amazing beach of Saranda!"
               label="Saranda"
```

Comment:
I would recommend creating a hard-coded data.json, which saves you code in Cards.js and makes everything clearer.

Reply:
Totally agree with Dajana on this one! Create a `data.json` file with an array with objects and map over it to create your cards.